cmd/cofidectl/cmd/down.go

```diff
@@ -39,12 +46,21 @@ func (d *DownCommand) DownCmd() *cobra.Command {
 				return err
 			}
 
-			statusCh, err := provision.TearDown(cmd.Context(), ds, kubeCfgFile)
+			tearDownOpts := provisionplugin.TearDownOpts{
+				KubeCfgFile: kubeCfgFile,
+				TrustZones:  opts.trustZones,
+			}
+			statusCh, err := provision.TearDown(cmd.Context(), ds, &tearDownOpts)
 			if err != nil {
 				return err
 			}
-			return statusspinner.WatchProvisionStatus(cmd.Context(), statusCh, false)
+			return statusspinner.WatchProvisionStatus(cmd.Context(), statusCh, opts.quiet)
 		},
 	}
+
+	f := cmd.Flags()
+	f.BoolVar(&opts.quiet, "quiet", false, "Minimise logging from uninstallation")
+	f.StringSliceVar(&opts.trustZones, "trust-zone", []string{}, "Trust zones to uninstall, or all if none is specified")
```

Comment:
Does this also work as `--trustzones tz1,tz2,tz3` etc?

Reply:
Yes

Reply:
If you can think of a better way to word the help message here I'm open to it.